content/home_content.php

```diff
@@ -13,8 +13,6 @@
 <br>
 <h1>Upcoming Events:</h1>
 <ul>
-  <li>USD FIRST WA Supporters Workshop - May 19th, 2016</li>
-  <br>
   <li>Hockinson Fun Days - June 4, 2016</li>
   <br>
```

Comment:
NOTE: This comment should not hold up the merging of this...Just a fun fact:
The HTML in this file is invalid, the only child tags allowed under a `ul` or a `ol` should only be `li`s.
Typically, spacing between list items is handled with css (ex. `padding: 2rem 1rem`), as means to not add markup just for styling. 
If you feel as a developer that you need to use `br`s I would recommend moving them INSIDE of the `li`'s...although you may need to adjust and styling you *may* have against the li tag. (Do not know if you do tho)